internal/config/package.go

```diff
@@ -0,0 +1,33 @@
+package config
```

Comment:
would this file be better named `pkg.go` ?

Reply:
(side note: I wanted to name the type `package` but that's a go keyword)